[16.0][ADD] survey_building_use_section

unaiberis opened the pull request:
Change from survey_building_type_section and res_partner_building_type_section

unaiberis committed:
[ADD] survey_building_use_section and res_partner_building_use_section. Change from survey_building_type_section and res_partner_building_type_section

anajuaristi commented on res_partner_building_use_section/views/res_partner_views.xml:
```diff
@@ -18,7 +18,8 @@
-                            <field name="risk_use" />
+                            <field name="risk" />
+                            <field name="section_use" />
                             <field name="superficie" />
```
falta da:
``<field name="evacuation_height" /> ``

anajuaristi commented on survey_building_use_section/models/res_partner.py:
```diff
@@ -7,5 +7,9 @@ class ResPartner(models.Model):
+        copy=False,
     )
+    building_use_id = fields.Many2one("building.use", string="Building Use")
+    is_industrial = fields.Boolean(string="Industrial")
```
Hauek bixek form bistan falta die. Ez die ikusten
```
                            <field name="is_industrial"/>
                            <field name="certification_text"/>
``

unaiberis committed:
[IMP] Add 3 fields

unaiberis committed:
[IMP] evacuation_height, section_use, is_industrial, certification_text

unaiberis committed:
[IMP] manifest and README

unaiberis committed:
[IMP] translations

unaiberis committed:
[IMP] Improve translations

unaiberis committed:
[IMP] translation

unaiberis committed:
[IMP] add fields

unaiberis committed:
[IMP] add fields

unaiberis committed:
[IMP] add fields

unaiberis committed:
[IMP] add fields

unaiberis committed:
[IMP] Add normativas_ids and _compute_normativas_ids

unaiberis committed:
[IMP] is_industrial change from res_partner to building_use

unaiberis committed:
[IMP] _filter_questions_of_normative

unaiberis committed:
[IMP] Add notes to matrix_row_ids

unaiberis committed:
[IMP] Risk/User to Risk

unaiberis committed:
[IMP] Put loggers

unaiberis committed:
[IMP] logging

unaiberis committed:
[IMP] logging

unaiberis committed:
[IMP] logging

unaiberis committed:
[IMP] logging

unaiberis committed:
[IMP] logging

unaiberis committed:
[IMP] partner_id and remove domain

unaiberis committed:
[IMP] service_end_date field

unaiberis committed:
[IMP] action_start_survey Button

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] action_start_survey Button improve

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Write loggers

unaiberis committed:
[IMP] Remove readonly

unaiberis committed:
[IMP] Remove readonly

unaiberis committed:
[IMP] Change survey.user_input partner_id to start survey

unaiberis committed:
[IMP] Change survey.user_input partner_id to start survey

unaiberis committed:
[IMP] Change survey.user_input partner_id to start survey to Administrator

unaiberis committed:
[IMP] Remove survey_user_input_lines without normative

unaiberis committed:
[IMP] Simplifiy loggers

unaiberis committed:
[IMP] create lines from vals after filtering, not before

unaiberis committed:
[IMP] Add question filter in controller

unaiberis committed:
[IMP] Add question filter in controller

unaiberis committed:
[IMP] Add question filter in controller

unaiberis committed:
[IMP] access_data['survey_sudo'].question_and_page_ids

unaiberis committed:
[IMP] access_data['survey_sudo'].question_and_page_ids assignment improvement

unaiberis committed:
[IMP] access_data['survey_sudo'].question_and_page_ids assignment improvement

unaiberis committed:
[IMP] Remove unnecessary code

unaiberis committed:
[IMP] Remove error

unaiberis committed:
[IMP] Add question filter and improve errors

unaiberis committed:
Merge branch '16.0' into 16.0-add-survey_building_use_section

unaiberis committed:
[IMP] Add domain in building_to_inspect_id

unaiberis committed:
[FIX] Remove unnecessary building to inspect id domain

unaiberis committed:
[IMP] js to add Duplicate column

unaiberis committed:
[IMP] Add js

unaiberis committed:
[IMP] Add logs

unaiberis committed:
[IMP] Write start() to render from the beginning

unaiberis committed:
[IMP] Remove title and membership number

[IMP] Remove title and membership number

[IMP] Add title and membership number

[IMP] Add new fields

[IMP] Add new fields

[IMP] Remove fields

[IMP] Remove fields

[IMP] Remove fields

[IMP] Remove fields

[IMP] .po

[IMP] xmlid

[IMP] .po Translations

[IMP] Remove duplicated fields

[IMP] Improve fields relations

[IMP] Improve fields relations

[IMP] Remove duplicates

[IMP] .po Translations

[IMP] Matrix filter: If the question doesn't have any question normatives, then show all matrix lines

[IMP] Add owl component

[IMP] Add owl component

[IMP] Add owl component

[IMP] Remove unnecessary comments

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Improvements

[IMP] Remove loggers

[IMP] Improve stop interval

[IMP] Remove error file

[IMP] Remove error file

[IMP] Improve th_article_width

[IMP] Improve code

[IMP] Improve code

[IMP] Improve code

[IMP] Improvements
